backend/test/forgotPassword.test.js

```diff
@@ -0,0 +1,125 @@
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const Admin = require('../app/models/Admin');
+const server = require('../index');
+
+chai.use(chaiHttp);
+
+before((done) => {
+  // Clearing the test database before testing
+  Admin.remove({}, () => {
+    done();
+  });
+});
+
+// Test suit for forgot password test cases
+describe('Forgot password tests', () => {
+  let forgotPasswordURL = null; // Initial forgot password url set to null
+
+  // Step 1 - We start by creating an admin
+  it('Creating an admin', (done) => {
+    const adminData = {
+      firstName: 'john',
+      lastName: 'wick',
+      email: 'john@wick.com',
+      password: '123asd',
+      contact: '+919090909090',
+      username: 'johnwick123',
+    };
+
+    chai
+      .request(server)
+      .post('/admin/superAdmin')
+      .send(adminData)
+      .end((err, res) => {
+        expect(err).to.equal(null);
+        expect(res.status).to.equal(200);
+        expect(res.body.firstName).to.equal(adminData.firstName);
+        expect(res.body.lastName).to.equal(adminData.lastName);
+        expect(res.body.isSuperAdmin).to.equal(true);
+        done();
+      });
+  });
+
+  // Step 2 - We log in the admin using the above credentials
+  it('Logging in with the old password', (done) => {
```

Comment:
Why do we need to login with old password?

Reply:
Just to make sure that our Admin is created after we complete `Step 1`

Reply:
Response of the API should be good enough for that. isn't it?

Reply:
Okay done.